client/src/app/dashboard/table-all/table-all.component.ts

```diff
@@ -44,9 +52,12 @@ export class TableAllComponent implements OnInit {
     @Input() compType: string;
 
 
-    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'action'];
+    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'Details', 'action'];
+    columnsToDisplay: string[] = ['dockerId', 'labels'];
     dataSource: MatTableDataSource<Instance> = new MatTableDataSource<Instance>(this.dataArray);
+    data = new MatTableDataSource<Instance>();
     dialogResult: string;
+    expandedElement: number;
```

Comment:
the type of the expandedElement is not number but actually the table element, thus an instance.